src/pudl/metadata/sources.py

```diff
@@ -680,7 +680,7 @@
         "working_partitions": {
             # 2021 and later data is in XBRL.
             # 2006-2020 data is in monolithic CSV files, so any year means all years.
-            "years": sorted(set(range(2006, 2024))),
+            "years": sorted(set(range(2006, 2025))),
```

Comment:
We haven't integrated the 2024 data yet -- the only reason there's a handful of 2024 hours is because of the time shift.